pkg/ccl/spanconfigccl/spanconfigsqltranslatorccl/datadriven_test.go

```diff
@@ -183,6 +203,37 @@ func TestDataDriven(t *testing.T) {
 					mutable.SetPublic()
 				})
 
+			case "protect":
+				var recordID string
+				var protectTS int
+				d.ScanArgs(t, "record-id", &recordID)
+				d.ScanArgs(t, "ts", &protectTS)
+				target := spanconfigtestutils.ParseProtectionTarget(t, d.Input)
+
+				jobID := tenant.JobsRegistry().MakeJobID()
+				require.NoError(t, tenant.ExecCfg().DB.Txn(ctx,
+					func(ctx context.Context, txn *kv.Txn) (err error) {
+						require.Len(t, recordID, 1,
+							"datadriven test only supports single digit record IDs")
+						recID, err := uuid.FromBytes([]byte(strings.Repeat(recordID, 16)))
+						require.NoError(t, err)
+						rec := jobsprotectedts.MakeRecord(recID, int64(jobID),
+							hlc.Timestamp{WallTime: int64(protectTS)}, nil, /* deprecatedSpans */
+							jobsprotectedts.Jobs, target)
+						return tenant.ProtectedTimestampProvider().Protect(ctx, txn, rec)
+					}))
+
+			case "release":
+				var recordID string
+				d.ScanArgs(t, "record-id", &recordID)
+				require.NoError(t, tenant.ExecCfg().DB.Txn(ctx,
+					func(ctx context.Context, txn *kv.Txn) error {
+						require.Len(t, recordID, 1,
+							"datadriven test only supports single digit record IDs")
```

Comment:
[nit] Just to make these datadriven tests clearer ("protect record-id=2 ts=2" for e.g.), I think using characters here would be better.